CMakePresets.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": 6,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 23,
```

Comment:
This is inconsistent with CMakeLists.txt, which requires 3.21.

Reply:
Note also that version 6 is new in version 3.25. Version 3 is new in version 3.21. (See https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html)